mlir/lib/Conversion/AIRRtToNpuPass.cpp

```diff
@@ -1251,6 +1258,145 @@ struct AIRRtToNpuPass : public impl::AIRRtToNpuBase<AIRRtToNpuPass> {
     }
   }
 
+  void insertNpuWrite32ForTrace(ModuleOp module, int64_t trace_size,
```

Comment:
This needs a unit test in `mlir/test/Conversion/AIRRtToNpu`

Reply:
added [mlir/test/Conversion/AIRRtToNpu/generate_trace_write32.mlir](https://github.com/Xilinx/mlir-air/pull/592/files#diff-bc62766234896565f7f875c3129091accef48afa0440e3893454e610d4b6639c)